src/stories/DropDownButton.js

```diff
@@ -36,6 +36,18 @@ storiesOf('Core/DropDownButton', module)
             <div>Item</div>
             <div>Item</div>
           </DropDownButton>
+          <DropDownButton
+            mx={2}
+            maxHeight={120}
+            label={<div>Fixed Height</div>}
```

Comment:
Fixed height and max height are subtly different, perhaps changing this to `Max Height` is better?

Reply:
True, I will change it.